pkg/cli/searcher.go

```diff
@@ -802,6 +878,44 @@ func printCVETableHeader(writer io.Writer, verbose bool, maxImgLen, maxTagLen, m
 	table.Render()
 }
 
+func printReferrersTableHeader(writer io.Writer, maxArtifactTypeLen int) {
+	table := getReferrersTableWriter(writer)
+
+	table.SetColMinWidth(refArtifactTypeIndex, digestWidth)
```

Comment:
Is this minimum size ok? It's for a different field. Just checking this is not a mistake.